src/Symfony/Component/Console/Command/Command.php

```diff
@@ -572,6 +572,8 @@ public function getSynopsis($short = false)
      * Add a command usage example.
      *
      * @param string $usage The usage, it'll be prefixed with the command name
+     *
+     * @return Command
```

Comment:
it could be even more strict and state `@return $this` as not a `Command` class instance is return but the instance itself is always returned (for ref. https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#keyword point 13 vs. 15).